packages/@react-native-windows/tester/src/js/examples-win/LegacyTests/TextBackgroundColorTestPage.tsx

```diff
@@ -60,16 +58,6 @@ export function TextBackgroundColorTestPage() {
       </Text>
       </View>
 
-      <TreeDumpControl
-        style={{
-          height: 150,
-          width: 500,
-          margin: 10,
-        }}
-        dumpID={'TextColorBackground'}
-        uiaID={'text-color-background-view'}
-        additionalProperties={['TextHighlighters']}
```

Comment:
how are you going to ask for this and other additional properties to the tree dumper?

Reply:
Can send arbitrary data over IPC channel. Will add extra param for it.